src/main/java/org/mifos/processor/bulk/camel/routes/BatchStatusRoute.java

```diff
@@ -65,7 +65,7 @@ public void configure() throws Exception {
                 .process(exchange -> {
                     BatchDTO batchSummary = exchange.getIn().getBody(BatchDTO.class);
 
-                    long percentage = (batchSummary.getCompleted()/batchSummary.getTotalTransactions())*100;
+                    long percentage = (batchSummary.getSuccessful()/batchSummary.getTotal())*100;
```

Comment:
why this is long? are we ignoring the decimal part ?  looks like this division is also an integer division. Is this expected behaviour?

Reply:
We don't need a decimal value, its just a percentage threshold check.